### PR TITLE
support configuration of node package manager in serverless.yml

### PR DESCRIPTION
**Pull request by theburningmonk**
as suggested by @omryhay1 in #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ provider:
 custom:
   lumigo:
     token: <YOUR TOKEN GOES HERE>
+    nodePackageManager: <node or yarn>
```
Comment (efimk-lu) [inline README.md]: npm or yarn

Comment (theburningmonk) [reply README.md]: lol d'oh

Comment (theburningmonk) [reply README.md]: fixed

**Comment by theburningmonk** [comment]
@efimk-lu yes, because a user would probably have both, but they actually want to choose Yarn.

**Pull request merged**